### ISSUE TITLE
Make loss debgging easier

### ISSUE THREAD

**Issue by josephdviviano** [GFNOrg/torchgfn#146]
https://github.com/GFNOrg/torchgfn/blob/d9ca55815c43e129fad95fdcb408280c90b19963/src/gfn/gflownet/flow_matching.py#L98

There should be a debug flag you can use to inspect the logits going into the loss and other issues like this

**Comment by josephdviviano** [comment]
I think the debugger tool is the better way of doing this, now that I am old and wise.
